tapis-project/tapis-ui
# Bug/TUI-21 - Fix bug with sidebar DOM

## Overview: ##
This PR should fix the issues with the app sidebar.

## Related Github Issues: ##

* [TUI-21](https://github.com/tapis-project/tapis-ui/issues/21)

## Summary of Changes: ##
Added Bootstrap/Reactstrap packages. Setup sidebar navigation for login, system, and UI pattern components.

## Testing Steps: ##
1. Run npm ci
2. Check the navigation pane so that it looks like the images below
3. The Systems button will not display unless you're logged in

## UI Photos:
<img width="1676" alt="Screen Shot 2021-03-29 at 1 32 49 PM" src="https://user-images.githubusercontent.com/29575979/112883820-5f198680-9094-11eb-86fd-6b481e309de6.png">
<img width="1677" alt="Screen Shot 2021-03-29 at 1 32 29 PM" src="https://user-images.githubusercontent.com/29575979/112883846-63de3a80-9094-11eb-9504-ade976aff3e7.png">
<img width="1673" alt="Screen Shot 2021-03-29 at 1 32 42 PM" src="https://user-images.githubusercontent.com/29575979/112883864-6771c180-9094-11eb-970c-0990a638b1c4.png">
<img width="1641" alt="Screen Shot 2021-03-29 at 1 32 19 PM" src="https://user-images.githubusercontent.com/29575979/112883884-69d41b80-9094-11eb-8bdf-724988a77140.png">

## Notes:
The grey bar on the left does not extend to the bottom of the page. (This is still an issue in CepV2)
